tests/rptest/services/kgo_verifier_services.py

```diff
@@ -7,12 +7,14 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0
 
+from __future__ import annotations
```

Comment:
Need to allow a class method to accept an argument of the same type as the class itself, e.g.:

```
class Foo:

  def takes_foo(self, another_foo: Foo):
     ...
```